.prettierrc

```diff
@@ -2,5 +2,6 @@
   "printWidth": 100,
   "singleQuote": true,
   "trailingComma": "none",
-  "arrowParens": "avoid"
+  "arrowParens": "avoid",
+  "proseWrap": "always"
```

Comment:
Note: Without this line Prettier wouldn't fix my markdown, and at the same time `markdownlint` would complain about my README having too long lines. Adding this also reformatted the README though, hence the changes below.